docs/dev/compile/Dependencies.rst

```diff
@@ -104,23 +117,27 @@ through your package manager, if possible.
 Building 32-bit Binaries
 ~~~~~~~~~~~~~~~~~~~~~~~~
 If you want to compile 32-bit DFHack on 64-bit distributions, you'll need the
-multilib development tools and libraries:
+multilib development tools and libraries.
+
+On Debian-based distributions, these are:
 
 * ``gcc-multilib`` and ``g++-multilib``
-* If you have installed a non-default version of GCC - for example, GCC 4.8 on a
-  distribution that defaults to 5.x - you may need to add the version number to
-  the multilib packages.
 
-  * For example, ``gcc-4.8-multilib`` and ``g++-4.8-multilib`` if installing for GCC 4.8
-    on a system that uses a later GCC version.
-  * This is definitely required on Ubuntu/Debian, check if using a different distribution.
+  * If you have installed a non-default version of GCC - for example, GCC 4.8 on a
+    distribution that defaults to 5.x - you may need to add the version number to
+    the multilib packages.
+
+    * For example, ``gcc-4.8-multilib`` and ``g++-4.8-multilib`` if installing for GCC 4.8
+      on a system that uses a later GCC version.
+    * This is definitely required on Ubuntu/Debian, check if using a different distribution.
+
+  * Note that installing a 32-bit GCC on 64-bit systems (e.g. ``gcc:i386`` on
```

Comment:
```suggestion
  * Note that installing 32-bit GCC on 64-bit systems (e.g. ``gcc:i386`` on
```

Reply:
Disagree - it's similar to "an installation of GCC"

Reply:
I think it's actually more like "installing a 32-bit Windows"

Reply:
You can have more than one installation of GCC on a system. You can't have more than one installation of Windows on a system (ok, disregarding VMs, dual-booting, etc., but those aren't relevant in the context of compiling DFHack on _one_ system). I can reword this, though - maybe "installing a 32-bit version of GCC"

Reply:
That sounds fine in my ear.